src/test/java/com/jcabi/jdbc/JdbcSessionITCase.java

```diff
@@ -40,6 +46,7 @@
  * Integration case for {@link JdbcSession}.
  * @author Yegor Bugayenko (yegor@teamed.io)
  * @version $Id$
+ * @checkstyle StringLiteralsConcatenationCheck (300 lines)
```

Comment:
@mkordas there's no other way to do it: if I concatenate with ``StringBuilder`` it also complains. If I leave it in one line it's more than 80 chars.